source/User_Guide/Marketing_Campaigns/index.html

```diff
@@ -1,55 +1,93 @@
 ---
 layout: page
 weight: 100
-title: Marketing Campaigns
+title: Overview
 navigation:
   show: true
 seo:
-  title: Marketing Campaigns
+  title: Marketing Campaigns Overview
   override: true
-  description: With SendGrid Marketing Campaigns, you will be able to load in contacts, create segments, create and send campaigns, view your stats, and so much more.
+  description: SendGrid Marketing Campaigns overview and video tutorials.
 ---
 
-<iframe src="https://player.vimeo.com/video/146014290" width="700" height="400" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
-<p>
-  With <a href="{{site.marketing_campaigns_url}}">SendGrid Marketing Campaigns</a>, you will be able to load in contacts,
-  create segments, create and send campaigns, view your stats, and so much more.
-</p>
+  With <a href="{{site.marketing_campaigns_url}}">SendGrid Marketing Campaigns</a>, you will be able to add contacts,
+  create segments, create and send campaigns, view your stats, and so much more. 
+
+  <br></br>
+  <iframe src="https://player.vimeo.com/video/185514373" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+
+  {% anchor h2 %}
+  Marketing Campaigns features and video tutorials
+  {% endanchor %}
 
 <p><strong><a href="https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/getting_started.html">Get started creating and sending your campaign</a></strong> to modify content, choose which lists to send to, add a suppression
-group, add categories for stats tracking, and then send right away, or schedule your email to send at another time.</p>
+group, add categories for stats tracking, and then send right away, or schedule your email to send at another time.</p> 
+<p><iframe src="https://player.vimeo.com/video/221496578" width="700" height="400" frameborder="0" allowfullscreen=""></iframe></p> 
+
 
 <p><strong><a href="{{root_url}}/User_Guide/Marketing_Campaigns/senders.html">Sender Identities</a></strong> -
 Add your sender identity in the <a href="{{site.marketing_campaigns_url}}/senders">sender management section</a>.
-This is the from name, from email address, reply-to email address, and physical address that are sent with your campaigns.
+This is the from name, from email address, reply-to email address, and physical address that are sent with your campaigns. 
 </p>
+<p><iframe src="https://player.vimeo.com/video/120703745" width="700" height="400" frameborder="0" allowfullscreen=""></iframe></p>
 
 <p><strong><a href="{{root_url}}/User_Guide/Marketing_Campaigns/custom_fields.html">Custom Fields</a></strong> - Set up your <a href="{{site.marketing_campaigns_url}}/custom_fields">custom fields</a> so that you can upload
 contact-specific data. This allows us to automatically generate substitution tags for your emails and unlocks the ability to segment your contacts as well.
```

Comment:
I feel like these two sentences could be combined into one:
Set up your custom fields so that you can upload contact-specific data, automatically generate substitution tags for your emails, and segment your contacts.